go/vt/vtctl/reparentutil/util.go

```diff
@@ -88,18 +88,23 @@ func ElectNewPrimary(
 
 	// candidates are the list of tablets that can be potentially promoted after filtering out based on preliminary checks.
 	candidates := []*topodatapb.Tablet{}
+	var reasonsToInvalidate string
```

Comment:
I don't think that performance is a real concern here, but a StringBuilder would a little more efficient. Or even a string slice where we later use strings.Join with a newline might be slightly nicer. This is awfully nitty though so totally up to you -- whatever you prefer.

Reply:
Done